contrib/tracefile-analyser.php

```
@@ -15,7 +16,20 @@
    +----------------------------------------------------------------------+
    | Authors: Derick Rethans <derick@xdebug.org>                          |
    +----------------------------------------------------------------------+
- */
+*/
+
+
+/*
+   php.ini additions for operation:
+      xdebug.profiler_enable = 1
+      debug.trace_format = 1
+      xdebug.collect_includes = 1   (optional)
+*/
+
+
+error_reporting(0); // suppress notices and warnings which disrupt terminal output
```

Instead of hiding errors, I'd rather see them fixed.